ingest/validation/validate_metadata.py

```diff
@@ -915,8 +915,24 @@ def process_metadata_row(metadata, convention, line):
     row_info = dict(itertools.zip_longest(metadata_names, line))
     processed_row = {}
     for k, v in row_info.items():
-        # for metadata not in convention, no need to process
+        # for metadata not in convention, check numeric metadata for valid values
         if k not in convention["properties"].keys():
+            type_index = metadata.headers.index(k)
+            k_type = metadata.annot_types[type_index]
+            if k_type == "numeric":
+                k_numeric = type(v) == int or type(v) == float
```

Comment:
Faster, and more concise:

```suggestion
                k_numeric = type(v) in [int, float]
```

For reference:
```
>>> foo = 1
>>> bar = 1.2
>>> baz = "hi"
>>> type (foo) in [int, float]
True
>>> type (bar) in [int, float]
True
>>> type (baz) in [int, float]
False
```

Reply:
Thanks! That is much more readable :)